NFive.SDK.Core.sln.DotSettings

```diff
@@ -75,6 +75,7 @@
     &lt;/Entry&gt;&#xD;
   &lt;/TypePattern&gt;&#xD;
 &lt;/Patterns&gt;</s:String>
+	<s:Boolean x:Key="/Default/CodeStyle/CSharpUsing/AddImportsToDeepestScope/@EntryValue">True</s:Boolean>
```

Comment:
Not sure I really see any benefit to deep scopes here, it just generates a lot of change noise.